### DISCUSSION TITLE
Black screens on the Mia-Platform Console

### DISCUSSION THREAD

**Discussion by lucascanna** [mia-platform/community#258] [Ideas]
### Which area of this project could be most improved?

Console

### Description

Hi,

I would like to suggest you a small but important improvement.

Every time you need to write some code from the console (e.g. configuration, Rego policies etc..) I would avoid black screens like the one below:
<img width="1413" alt="Screenshot 2023-05-19 at 17 32 45" src="https://github.com/mia-platform/community/assets/5745481/232e8bd0-c684-4bc2-830d-85a6715d9916">

You could improve a lot the Developer Experience by adding a simple snippet of example.

Thanks!

**Comment by fredmaggiowski** [comment]
This is something we can easily do @albertotessarotto let's keep track of the feedback

**Comment by albertotessarotto** [comment]
I agree, finding a blank space can be disorienting to start with. Thank you for the feedback, we can think about

**Comment by nicola88** [comment]
@albertotessarotto news on this or we can close it?
